parts/kubernetes/agentpool/kubelet.service

```diff
@@ -0,0 +1,53 @@
+[Unit]
```

Comment:
An equivalent file for a kubelet.service already exists here:

https://github.com/Azure/acs-engine/blob/master/parts/kuberneteskubelet.service

Let's keep this centralized so that we aren't continually flipping flags in multiple places.